css/src/components/dropdown.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .Dropdown-inputWrapper {
-  border-bottom: var(--border);
+  border-bottom: 1px solid #c5c5c5;
```

Comment:
Please use CSS variable for particular color

Reply:
Done